content/docs/fundamentals/converting/resolving-errors@ko.md

```diff
@@ -0,0 +1,350 @@
+
+이전과 마찬가지로 페이지를 **새로고침**하고 오류가 사라졌는지 확인합니다.
+
+## 외부 스타일시트 교체
+
+다음의 오류는 스타일시트 사용과 관련되어 있습니다.
+
+<pre class="error-text">
+'link rel=stylesheet for fonts' 태그에 있는 'href' 속성이 잘못된 값인 'base.css'로 설정되어 있습니다.
+</pre>
+
+특히 이 오류는 `<head>` 태그에 있는 다음과 같은 스타일시트 링크 태그와 관련되어 있습니다.
+
+```html
+<link href="base.css" rel="stylesheet" />
+```
+
+문제는 외부 스타일시트를 참조한 데서 발생합니다. 문서를 가능한 한 빨리 로드하기 위해 AMP에는 외부 스타일시트를 포함할 수 없습니다. 대신 모든 스타일시트 규칙은 `<style amp-custom></style>` 태그를 사용하여 AMP 문서에 인라인으로 추가되어야 합니다.
+
+```html
+<style amp-custom>
+
+/* The content from base.css */
+
+</style>
+```
+
+이제 오류를 해결해 보겠습니다.
+
+1.  `<head>`에서 외부 스타일시트를 가리키는 `<link>` 태그를 **삭제**하고 이를 인라인 `<style amp-custom></style>` 태그로 교체합니다. 스타일 태그의 `amp-custom` 속성은 필수입니다.
+2. [`base.css`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.css) 파일에 있는 모든 스타일을 **복사**하여 `<style amp-custom></style>` 태그에 붙여넣습니다.
+
+다시 한번 페이지를 **새로고침**하고 스타일시트 오류가 사라졌는지 확인합니다.
+
+{% call callout('참고', type='note') %}
+인라인 스타일링이 필요할 뿐 아니라 모든 스타일링 정보의 파일 크기가 50KB로 제한됩니다. AMP 페이지에 CSS를 인라인 처리하기 전에 [SASS](http://sass-lang.com/)와 같은 CSS 사전 처리기를 사용하여 CSS를 축소해야 합니다.
+{% endcall %}
+
+{% call callout('중요', type='caution') %}
+전체 AMP 문서에 대해 스타일 태그를 1개만 사용할 수 있습니다. AMP 페이지에서 참조하는 외부 스타일시트가 여러 개인 경우 이러한 스타일시트를 하나의 규칙 세트로 조합해야 합니다. AMP에서 어떤 CSS 규칙이 유효한지 알아보려면 [지원되는 CSS](/ko/docs/design/responsive/style_pages.html)를 확인하세요.
+{% endcall %}
+
+## 타사 자바스크립트 제외
+
+CSS를 인라인 처리하면 스타일시트를 AMP로 비교적 쉽게 다시 작성할 수 있지만 자바스크립트에서는 그렇지 않습니다.
+
+<pre class="error-text">
+'script' 태그는 특정 형식 이외에는 허용되지 않습니다.
+</pre>
+
+일반적으로 AMP에서 스크립트는 두 가지 주요 요구사항을 따르는 경우에만 허용됩니다.
+
+1.  모든 자바스크립트는 비동기식이어야 합니다(즉, 스크립트 태그에 `async` 속성을 포함해야 함).
+2.  AMP 라이브러리 및 페이지에 있는 AMP 구성요소용으로 자바스크립트를 사용해야 합니다.
+
+이와 같은 요구사항으로 인해 다음에 명시된 경우를 제외하고는 AMP에서 모든 사용자 생성/타사 자바스크립트의 사용이 사실상 배제됩니다.
+
+{% call callout('참고', type='note') %}
+사용자 생성/타사 스크립트 사용 제한의 유일한 예외는 다음과 같습니다.
+
+1.  스크립트가 페이지에 메타데이터를 추가하거나 AMP 구성요소를 구성합니다. 이러한 스크립트에는 `application/ld+json` 또는 `application/json` 유형 속성이 포함됩니다.
+2.  스크립트가 iframes에 포함되어 있습니다.  iframe에 자바스크립트를 포함하는 일은 최후의 수단으로 생각해야 합니다. 가능하다면 자바스크립트 기능은 [AMP 구성요소](/ko/docs/reference/components.html)를 사용하여 대체해야 합니다. 다음 섹션에서 첫 번째 AMP 구성요소를 살펴보겠습니다.
+{% endcall %}
+
+외부 [`base.js`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.js) 파일을 열어 보세요. 무엇이 표시되나요? 이 파일에는 자바스크립트 코드가 없어야 하며 다음과 같은 정보의 주석만 포함되어 있어야 합니다.
+
+```javascript
+/*
+
+이 외부 자바스크립트 파일은 의도적으로 비어 있습니다.
+
+이 파일은 외부 자바스크립트 파일 사용과 관련된
+AMP 유효성 검사 오류를 보여주기 위해 사용됩니다.
+
+*/
+```
+
+이 외부 자바스크립트 파일은 웹사이트의 기능 구성요소가 아니므로 참조를 완전히 삭제할 수 있습니다.
+
+문서에서 다음 외부 자바스크립트 참조를 **삭제**합니다.
+
+```html
+<script type="text/javascript" src="base.js"></script>
+```
+
+이제 페이지를 **새로고침**하고 스크립트 오류가 사라졌는지 확인합니다.
+
+## AMP CSS 상용구 포함
+
+다음 오류는 누락된 상용구 코드를 참조합니다.
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).